demos/input/index.html

```diff
@@ -150,6 +173,18 @@
                      data-value="35.32"></div>
             </div>
         </div>
+        <div class="grid-col-6">
+            <div class="form-group">
+                <label>Datalist 2</label>
+                <div data-aura-component="input@husky"
```

Comment:
Did you also think about implementing that in a separate component? The `input` is already quite big 😕